recipes/pyebsdindex/meta.yaml

```diff
@@ -48,6 +48,7 @@ test:
 about:
   home: https://pyebsdindex.readthedocs.io
   license_file: License
+  license: LicenseRef-CUSTOM
```

Comment:
According to upstream it is Pubic Domain: https://github.com/USNavalResearchLaboratory/PyEBSDIndex/blob/main/License#L4

Also, if you choose a custom license it should be `LicenseRef-PyEBSDIndex`.

BTW, Public Domain is not in SPDX b/c it is, in a way, not a license, if you chose that it should be `LicenseRef-Public-Domain`.